storybook/stories/click-outside/README.md

```diff
@@ -0,0 +1,17 @@
+## useClickOutside Hook
```

Comment:
In the docs I think its best to call the hook Click Outside Hook, we changed this on other docs.

https://react-browser-hooks.netlify.com/?selectedKind=PageVisibility&selectedStory=Default&full=0&addons=1&stories=1&panelRight=1&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel